eng/Analyzers_ShippingRules.ruleset

```diff
@@ -1,41 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<RuleSet Name="Common diagnostic rules for all Roslyn Analyzer projects"
-         Description="This file contains diagnostic settings used by all Roslyn Analyzer projects. Projects that need specific settings should have their own rule set files that Include this one, and then make the necessary adjustments." 
-         ToolsVersion="14.0">
-
-  <Rules AnalyzerId="Microsoft.ApiDesignGuidelines.Analyzers" RuleNamespace="Microsoft.ApiDesignGuidelines.Analyzers">
-    <!-- ValidateArgumentsOfPublicMethods - only useful for libraries with supported public API surface, we don't have any -->
-    <Rule Id="CA1062" Action="None" />
-    <!-- CA1014: Mark assemblies with CLSCompliant -->
-    <Rule Id="CA1014" Action="None" />
```

Comment:
This rule has `isEnabledByDefaultInAggressiveMode: false`, so it's not necessary to have an explicit entry, so I dropped it.